samples/net/sockets/big_http_download/prj.conf

```diff
@@ -2,7 +2,8 @@
 CONFIG_NEWLIB_LIBC=y
```

Comment:
Typo in the commit msg `s/sickets/sockets/`

Reply:
fixed